docs/release-notes/8.4.asciidoc

```diff
@@ -1,6 +1,17 @@
 [[release-notes-header-8.4.0]]
 == 8.4
 
+[discrete]
+[[release-notes-8.4.3]]
+=== 8.4.3
+
+[discrete]
+[[bug-fixes-8.4.3]]
+==== Bug fixes and enhancements
+* Aligns the delete icon in the Add Rule Exception flyout ({pull}141365[#141365]).
+* Aligns the warning message title on the Rule details page with the warning icon  ({pull}140719[#140719]).
+* Fixes a bug that sometimes caused {elastic-endpoint} to stop running on Windows endpoints.
```

Comment:
If you want to link to a git issue, this is the one to link to https://github.com/elastic/endpoint/issues/29

Reply:
@ferullo sure, np! I was forgoing the link in case you wanted to reference a different issue or PR. We can use the same one we previously referenced though.

```suggestion
* Fixes a bug that sometimes caused {elastic-endpoint} to stop running on Windows endpoints (https://github.com/elastic/endpoint/issues/29[#29]).
```

Reply:
👍 The PR to address it is private, so that github link is still the best one to use.